lib/organisms/gravity-form/_style.scss

```diff
@@ -0,0 +1,50 @@
+.ln-o-gravity-form {
+
+  .hidden {
+    display: none !important;
+  }
+
+  &-header {
```

Comment:
I will say if is not used we should get ride of this file since we already have a base style for this one.